### PR TITLE
ID-396 Content changes for donor confirm identity screen

### PR DESCRIPTION
**Pull request by mohammedrmoj**
## Purpose

Changes to the how will you confirm identity page for donor. To implement Figma: https://www.figma.com/design/pEYIyMBoWCGAvTclJFMOsq/Paper-ID-ideas?node-id=7979-43641&node-type=frame&t=GBRaEfV1izMR9N49-0

## Approach

Markup changes

### CODE REVIEW

File: service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig [hunk 1]
```diff
@@ -5,220 +5,263 @@
...
+                    {% endif %}
                 </div>
-            {% endif %}
-        </div>
-        <details class="govuk-details">
-            <summary class="govuk-details__summary">
+                <details class="govuk-details">
+                    <summary class="govuk-details__summary">
             <span class="govuk-details__summary-text">
                 Other methods
```
Comment (davidbridgwood) [inline service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig:220]: in the figma it looks like this has been changed to a heading with some slightly different wording for the options.

Comment (mohammedrmoj) [reply service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig:220]: Well spotted. I didn't notice that. Not very good at spot the difference apparently.

Comment (mohammedrmoj) [reply service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig]: Done

Comment (davidbridgwood) [reply service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig]: Hey @mohammedrmoj, this still doesn't quite match the figma.

figma
vs
actual

**Comment by davidbridgwood** [comment]
theres also a [failing cypress test](https://github.com/ministryofjustice/opg-paper-identity/actions/runs/11802518032/job/32878588866?pr=297) - i'm not too familiar with cypress myself yet but looks like some changes are needed in [donor-journey.cy.ts](https://github.com/ministryofjustice/opg-paper-identity/blob/feature/ID-396-changes-donor-confirm-identity/e2e-tests/cypress/e2e/donor/donor-journey.cy.ts) to match your changes.

**Comment by mohammedrmoj** [comment]
@davidbridgwood All passing now

**Review by davidbridgwood — APPROVED** [review]

**Pull request merged**